experiments/misc/ilamb_conversion.jl

```diff
@@ -0,0 +1,389 @@
+# The purpose of this script is to produce NetCDF files compatible with ILAMB from
+# the diagnostics produced by the land simulation.
+#
+# To run this script, you can run
+# ```
+# julia --project=.buildkite experiments/misc/ilamb_conversion.jl <output_dir> <save_dir>
+# ```
+# in the terminal.
+
+import ClimaAnalysis
+import NCDatasets
+import Dates
+
+"""
+    struct ILAMBMapping
+
+Keep track of the short name, long name, conversion factor, and units for
+making a NetCDF file produced by ClimaDiagnostics compatible with ILAMB.
+"""
+struct ILAMBMapping
+    """Short name of the variable in the NetCDF file for ILAMB"""
+    short_name::String
+
+    """Long name of the variable in the NetCDF file for ILAMB"""
+    long_name::String
+
+    """Conversion factor from CliMA to ILAMB"""
+    conversion_factor::Float64
+
+    """Units of variable after applying conversion factor"""
+    units::String
```

Comment:
```suggestion
    ilamb_units::String
```
might be helpful to distinguish from clima units?